easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb

```diff
@@ -0,0 +1,64 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+name = 'Java'
+easyblock = 'ConfigureMake'
+version = '1.8_191-b26-OpenJDK'
+
+homepage = "https://openjdk.java.net/"
+description = """An open-source implementation of the Java Platform, Standard Edition"""
+
+toolchain = {'name': 'GCCcore', 'version': 'system'}
+
+source_urls = [
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/corba/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/jaxp/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/jaxws/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/langtools/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/jdk/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/nashorn/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/hotspot/archive/',
+]
+sources = [
```

Comment:
@edmondac Maybe include a comment how these were determined, and how they relate to the version you set?

Reply:
Done